cmd/login.go

```diff
@@ -114,7 +114,7 @@ Example:
 					return errors.New("Password is required")
 				}
 			}
-			restClient := client.NewRestClient(Config.Url, Config.Token, Config.APIVersion, logging.Verbose, Config.Cert)
+			restClient := client.NewRestClient(Config.Url, "", Config.APIVersion, logging.Verbose, Config.Cert)
```

Comment:
This should be empty since login with username and password should start without any token.